src/doc/gui_manual/Quantitative/LineSampler.rst

```diff
@@ -0,0 +1,340 @@
+that consists of one or more "channels" over which the data is
+sampled. For each array the orientation of the plane can be
+defined. Whereas for each channel its orientation within the plane and
+the sampling type and spacing can be defined. For instance, the
+sampling can be a series of lines through the data, or a series of
+points integrated over time.
+
+Once defined, the operator will produce the appropriate geometry and
+sampling regardless of the window dimension. That is for a 3D window
+each channel will be drawn showing its 3D position. Similarly for a 2D
+window for the Y=0 or the Phi=0 plane. For 1D a window the results
+will be displayed as a curve plot in the same way a Lineout curve plot
+would be drawn.
+
+Unique and critical to using the LineSampler operator is that it
+should be defined in each window for each plot. Further the "Apply to
+all windows" and "Apply operators to all plots" should be checked. As
+such there is some critical setup that must be done.
+
+The genesis of the Line Sampler operator is from plasma physics and
+the synthetic diagnostics performed in fusion simulations in order to
+compare against experimental results. As such, the nomenclature is
+based on this usage.
+
+LineSampler operator
+~~~~~~~~~~~~~~~~~~~~
+
+Before using the LineSampler operator in main plot window set the
+Apply to "active window" and uncheck "Apply operators to all plots."
+This setting is critical for setting up all of the plots that will use
+the LineSampler operator. Once all of the plots are set up, then, and
+only then should the settings be changed.
+
+Create a 3D view of the data showing the channels - First, create a
+translucent Pseudocolor plot of the data for reference. Next, clone
+the plot and add the LineSampler Operator to the plot. Open the
+LineSampler operator attribute window.
+
+Main tab
+""""""""
+
+Select the "Main" tab, :numref:`Figure %s <LineSampler_MainTab>` to
+set up the attributes accordingly.
+
+1. Mesh geometry, which can be Cartesian, Cylindrical or Toroidal,
+   determines how the sampling will be done. The most notable
+   difference between Cylindrical and Toroidal is coordinate ordering:
+   (r, phi, z) vs (r, z, phi).
+
+2. The user can set up an array configuration specifying the geometry
+   or manually read in a channel configuration file. The "Geometry"
+   and "List" tabs reflect those respective choices.
+
+3. The boundary of the sampling can be based on the data file or using
+   a manually specified wall file. The wall file reflects the boundary
+   in a 2D slice of the data in the Y=0 or the Phi=0 plane. Any
+   sampling outside of the boundary will be disregarded, that is the
+   sample ray will be clipped.
+
+   Note currently the wall file format is specific to D3D fusion
+   files. The coordinates are displayed in the text box.
+
+4. When using the LineSampler operator, one will need to check the
+   "Apply operators to all plots" option. So that all of the
```

Comment:
remove the period, eg `option so that all of the`

Reply:
fixed